### PR TITLE
Add mentoring guidelines

### PR DESCRIPTION
**Pull request by tom-mullen**
Why:

* As our business grows, it's important that we not only make new
recruits comfortable, but also help them to learn about how we do things
and what we believe in

This change addresses the need by:

* Creating a document that both mentors and mentees can refer to during
the onboarding process

### CODE REVIEW

File: handbook/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+- **Discuss the fundamentals** such as Git (FF merges), linting and project setup
+
+- Explain how to **efficiently manage time** to create a sustainable but
+  productive working practice
+
+- Walk through the quirks of our **remote-first culture**, the fascination with
+  `afk` and the importance of fluid conversation regular updates
+
+- Post a picture of some kittens to the `#random` channel
```
Comment (vitaliiprodan) [inline handbook/mentoring.md:34]: Most important thing 😂

**Review by vitaliiprodan — COMMENTED** [review]
Should we mention something about walking through cookbook key points before starting to work?

File: handbook/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+- Offer reassurance on **taking ownership for delivering code that works** and
+  how to ship it to production with confidence
+
+- **Review pull requests** and offer constructive advice on how to improve code
+  where possible
+
+- **Introduce existing team members** and share any key information about the
+  day-to-day running of the project you're working on
+
```
Comment (qborreda) [inline handbook/mentoring.md:47]: I would add something about not jumping straight into code, but think on edge cases or worst case scenarios. I used to call this **defensive programming**

Comment (tom-mullen) [reply handbook/mentoring.md:47]: Hey Q, although I agree that's valid, we don't really want to get into all the intricate details here. Let's think of where we might want to add that 'general approach to programming'

**Comment by tom-mullen** [comment]
@vitaliiprodan good point, I made and addition.

**Pull request merged**